proto/rill/admin/v1/api.proto

```diff
@@ -1896,6 +1896,7 @@ message UnsubscribeReportRequest {
   string organization = 1;
   string project = 2;
   string name = 3;
+  optional string email = 4 [(validate.rules).string.email = true]; // for non-rill recipients
```

Comment:
See other comment, I think this is problematic (and can be inferred from the token)